tests/phpunit/integration/Modules/Analytics_4Test.php

```diff
@@ -201,6 +202,108 @@ function( Request $request ) use ( $property_id, $webdatastream_id, $measurement
 		);
 	}
 
+	public function test_handle_provisioning_callback__gteSupport() {
+		$this->enable_feature( 'gteSupport' );
+		$account_id       = '12345678';
+		$property_id      = '1001';
+		$webdatastream_id = '2001';
+		$measurement_id   = '1A2BCD345E';
+		$account_id       = '123';
```

Comment:
While this test passes well, this `$account_id` variable is being duplicated, which I believe is unintentional.

I would suggest prefixing the Google Tag settings with `google_tag`, e.g. `google_tag_account_id`. Let me know what you think.

Reply:
Yikes, you're right that's unintentional. I've fixed this one and also added another get_scopes test for `gteSupport`. Cheers.